src/Main.java

```diff
@@ -0,0 +1,153 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+            // 1. zadatak
+        Scanner reader = new Scanner(System.in);
+        System.out.println("How many elements will contain your array?");
+        int index = reader.nextInt();
```

Comment:
formating should be better, sepparate blocks of code which logically should be separate